terraform/modules/tf_stream_alert_flow_logs/variables.tf

```diff
@@ -1,19 +1,29 @@
-variable "destination_stream_arn" {}
+variable "destination_stream_arn" {
+  type = "string"
```

Comment:
@chunyong-lin I have this in almost every module for clarity purposes, i'm going to leave it